cmake/fficonfig.h.in

```diff
@@ -13,7 +13,7 @@
 /* #undef C_ALLOCA */
 
 /* Define to the flags needed for the .section .eh_frame directive. */
-#define EH_FRAME_FLAGS "aw"
```

Comment:
Generally speaking, `#define` should not be systematically replaced by `#cmakedefine`. 

Only the macros having a corresponding variable set during CMake configuration should be defined using `#cmakedefine`.